packages/xo-server/src/xo-mixins/backups.js

```diff
@@ -139,22 +139,20 @@ const listPartitions = (() => {
   })
 
   return device =>
-    execa
-      .stdout('partx', [
-        '--bytes',
-        '--output=NR,START,SIZE,NAME,UUID,TYPE',
-        '--pairs',
-        device.path,
-      ])
-      .then(stdout =>
-        mapFilter(splitLines(stdout), line => {
-          const partition = parseLine(line)
-          const { type } = partition
-          if (type != null && !IGNORED[+type]) {
-            return partition
-          }
-        })
-      )
+    execa('partx', [
+      '--bytes',
+      '--output=NR,START,SIZE,NAME,UUID,TYPE',
+      '--pairs',
+      device.path,
+    ]).then(({ stdout }) =>
+      mapFilter(splitLines(stdout), line => {
+        const partition = parseLine(line)
+        const { type } = partition
+        if (type != null && !IGNORED[+type]) {
```

Comment:
explicit the test for `!IGNORED[+type]`

Reply:
This is not related to this PR, she's only upgrading `execa` to latest version and changing existing code to work.